auth0-training/labs-node-working-with-connections
# Fix/keyword replacement

## Testing instructions
1. Launch lab in Codespaces
2. Ensure the extension config pulls down the correct URIs for Allowed login and callback URLs
